java/test/org/openqa/selenium/remote/tracing/opentelemetry/TracerTest.java

```diff
@@ -49,6 +38,22 @@
 import org.openqa.selenium.remote.tracing.Tracer;
 import org.openqa.selenium.testing.UnitTests;
 
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.common.AttributeKey;
```

Comment:
Optimizing imports can probably be a different PR. It will help review the core changes easily.

Reply:
I agree, it's just IDEA did it automatically. :)

Reply:
@pujagani Now I have removed the change in imports.